doc/source/whatsnew/v1.0.0.rst

```diff
@@ -324,7 +324,7 @@ Numeric
 - :class:`DataFrame` flex inequality comparisons methods (:meth:`DataFrame.lt`, :meth:`DataFrame.le`, :meth:`DataFrame.gt`, :meth: `DataFrame.ge`) with object-dtype and ``complex`` entries failing to raise ``TypeError`` like their :class:`Series` counterparts (:issue:`28079`)
 - Bug in :class:`DataFrame` logical operations (`&`, `|`, `^`) not matching :class:`Series` behavior by filling NA values (:issue:`28741`)
 - Bug in :meth:`DataFrame.interpolate` where specifying axis by name references variable before it is assigned (:issue:`29142`)
--
+- Bug in :meth:`Series.var` not computing the right value with an integer series with missing values when ddof = 0 (:issue:`29128`)
```

Comment:
say nullable integer dtype

its not ``ddof=0``, rather not passing thru the ``ddof`` arg

Reply:
Done